test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -232,58 +231,34 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 	When("pod runs with the CPU load balancing runtime class", func() {
 		var smtLevel int
 		var testpod *corev1.Pod
-		var defaultFlags map[int][]int
 
-		getCPUsSchedulingDomainFlags := func() (map[int][]int, error) {
-			cmd := []string{"/bin/bash", "-c", "more /proc/sys/kernel/sched_domain/cpu*/domain*/flags | cat"}
-			out, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+		getCPUsSchedulingDomains := func() ([]string, error) {
```

Comment:
the name does not match the result anymore

Reply:
I have changed the name of the function to `getCPUswithLoadBalanceDisabled`